### ISSUE TITLE
"show points and/or lines" option not saved in the layout file

### ISSUE THREAD

**Issue by orsoromeo** [facontidavide/PlotJuggler#269]
First of all, thanks again for creating plotjuggler!

I am trying to plot multiple curves in the same plotjuggler plots that have the same name and an overlapping time series. This results in plotjuggler showing these different curves as if they were the same curve as in this figure:
![plotjuggler](https://user-images.githubusercontent.com/14978496/78117066-184f6d00-73fd-11ea-9dc7-458cb7171b47.png)

This is fine with me, as I can work around this problem by simply plotting the points rather than the lines: 
![plot2](https://user-images.githubusercontent.com/14978496/78117172-39b05900-73fd-11ea-8865-39bffe861129.png)
However, I wish I was able to save my specific "show points and/or lines" option in the layout file. which is not the case at the moment. This means that whenever I start plotjuggler from scratch and I load my layout file the plot will look like in the first figure. I will then have to right click on each individual subplot in order to only show the points. 
In conclusion, It would save me a lot of time if I could save in the layout file the fact that I only want to show the points and not the lines.

**Comment by facontidavide** [comment]
I will try to find some time to implement this

**Comment by facontidavide** [comment]
done
